### PR TITLE
P0 screens

### PR DESCRIPTION
**Pull request by jeetsdev**
Added UI Screens

- Landing page 
- Home page
- Explore page
- Bookmark page
- Profile page

image
image
image
image
image

[Preview](https://deploy-preview-1--curious-kitten-ce220a.netlify.app/)

### CODE REVIEW

File: src/pages/Home/Home.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+.navbar-link:hover{
+	transition: color .3s ease-in-out;
+	color: var(--primary-hover-color);
+}
+
+
+/* ! Content container here */
+
+.content-container {
+	margin-top: 80px;
```
Comment (rahulmistry751) [inline src/pages/Home/Home.css]: You can use relative length unit instead of absolute unit

File: src/App.css [hunk 2]
```diff
@@ -1,51 +1,80 @@
...
+	background-color: var(--primary-hover-color);
+}
+
+.btn-primary-outline:hover {
+	background-color: var(--primary-color);
+	color: white;
+}
+
+.btn-secondary:hover {
+	color: white;
```
Comment (rahulmistry751) [inline src/App.css]: suggestion:css variable can be used for hover font color

**Review by rahulmistry751 — APPROVED** [review]
Clean code and easily understandable

**Review by anjali1102 — APPROVED** [review]
pretty UI
good to go
liked how extract code to different files
code bad formatting

**Pull request merged**